dotNetInstallerLib/ProcessComponent.cpp

```diff
@@ -40,10 +40,7 @@ bool ProcessComponent::IsExecuting() const
 
 void ProcessComponent::Wait(DWORD /* tt */)

```

Comment:
Related, this method takes a timeout? We're supposed to default it to `INFINITE` and otherwise pass it in I think and use it in `WaitForSingleObject`.